index.html

```diff
@@ -243,7 +248,25 @@ <h3>LET’S CONNECT</h3>
         </div>
       </section>
       <!--End of about section-->
-
-
+       <!--Start of Contact Page-->
+       <section class="contact-page">
+        <div class="contact-block">
+            <div class="description">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!</p>
+            </div>
+            <form action="#">
+                <fieldset>
+                    <input type="text" placeholder="Lorem Ipsum">
```

Comment:
- [ ] It required that the name field accepts **up to 30 chars**, therefore kindly add `maxlength="30"` attribute to the name field.
- [ ] It required that user shouldn't be able to submit a form when one or more fields is empty, Therefore kindly add `required` attribute to the name fields.